### DISCUSSION TITLE
Can't deploy on firebase using pnpm

### DISCUSSION THREAD

**Discussion by dsl101** [pnpm/pnpm#8819] [Q&A]
This is likely a Google issue, but just wanted to check here too—during firebase deploy, I get this error:
```
i  functions: updating Node.js 20 (1st Gen) function authUserOnCreate(us-central1)...
Build failed: installing pnpm: (error ID: e462f501):
downloading pnpm: (error ID: 44d6210b):
fetching https://github.com/pnpm/pnpm/releases/download/v9.14.3/pnpm-linux-x64 returned HTTP status: 404; Error ID: d34422e2
```
There doesn't seem to be a release for the 9.14.3 tag—is that right? Should this work and it's just been removed for some reason? Or is it misconfigured in the google backend?

**Comment by dsl101** [comment]
There are related issues #8820 and [here](https://github.com/firebase/firebase-tools/issues/8004)

**Reply by zkochan** [reply]
I am sorry. It should be fixed now.
